Data/DataLoader_mask2.py

```diff
@@ -0,0 +1,63 @@
+from pyparsing import Optional
+import pytorch_lightning as pl
+from pytorch_lightning.utilities.types import TRAIN_DATALOADERS
+from torch.utils.data import DataLoader
+from pytorch_lightning.utilities.types import EVAL_DATALOADERS, STEP_OUTPUT, TRAIN_DATALOADERS, OptimizerLRScheduler
+import pandas as pd
+import numpy as np
+from Dataset import RoadMarkingDataset
+
+class RoadMarkingDataloader(pl.LightningDataModule):
+    '''
+    Dataloader class for the Mask2Former model
+
+    '''
+    def __init__(self, train_csv:str, val_csv:str, test_csv:str,batch_size:int , image_column:str, mask_column:str):
+        '''
+        Constructor for the class
+        '''
+        super().__init__()
+        self.train_csv = train_csv
+        self.val_csv = val_csv
+        self.test_csv = test_csv
+        self.batch_size = batch_size
+        self.image_column = image_column
+        self.mask_column = mask_column
+        self.train_ds = None
```

Comment:
you don't have to initialize train_ds, val_ds and test_ds with None here, you could use some dict and do something like stage_ds = dict() and then in setup replace self.train_ds with stage_ds['train']. It's better to use a data structure when you have to store multiple variables.